app/controllers/admin/custom_fields/hierarchy/items_controller.rb

```diff
@@ -63,6 +65,18 @@ def create
           respond_with_turbo_streams
         end
 
+        def destroy
+          # TODO: user persistence service
+          @hierarchy_item.destroy
```

Comment:
Since I merged the persistence service, I think you could already implement this change here.

Reply:
definitely, will merge dev in and update this.